lms/static/sass/course/wiki/_wiki.scss

```diff
@@ -278,30 +278,6 @@ section.wiki {
           color: $link-color;
           font-weight: bold;
           background-color: $gray-l4;
-
-          .icon-view,
-          .icon-home {
-            background-position: -25px 0;
-          }
-
-          .icon-edit {
-            background-position: -25px -25px;
-          }
-
-          .icon-changes,
-          .icon-time {
-            background-position: -25px -49px;
-          }
-
-          .icon-attachments,
-          .icon-file {
-            background-position: -25px -73px;
-          }
-
-          .icon-settings {
-            background-position: -25px -99px;
-          }
-
```

Comment:
Yep, just confirmed that these sprite-based icons have been replaced with Font Awesome font icons. Here's a screengrab from my sanbox of a course wiki:

![screen shot 2015-01-02 at 12 23 14 pm](https://cloud.githubusercontent.com/assets/163763/5597935/38727970-927a-11e4-8ac7-2f523b28da4a.png)